databases/redis-spotahome/terraform/gke-standard/main.tf

```diff
@@ -0,0 +1,71 @@
+#Copyright 2023 Google LLC
+
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+# google_client_config and kubernetes provider must be explicitly specified like the following.
+data "google_client_config" "default" {}
+
+# create private subnet
+module "network" {
+  source         = "../modules/network"
+  project_id     = var.project_id
+  region         = var.region
+  cluster_prefix = var.cluster_prefix
+}
+
+# [START gke_redis_spotahome_standard_private_regional_cluster]
+module "redis_cluster" {
+  source                   = "../modules/cluster"
+  project_id               = var.project_id
+  region                   = var.region
+  cluster_prefix           = var.cluster_prefix
+  network                  = module.network.network_name
+  subnetwork               = module.network.subnet_name
+
+  node_pools = [
+    {
+      name            = "pool-redis"
+      disk_size_gb    = 20
+      disk_type       = "pd-standard"
+      autoscaling     = true
+      min_count       = 1
+      max_count       = 2
+      max_surge       = 1
+      max_unavailable = 0
+      machine_type    = "e2-standard-2"
+      auto_repair     = true
+    }
+  ]
+  node_pools_labels = {
+    all = {}
+    pool-kafka = {
```

Comment:
**Question:** I see a few mentioned of "kafka". Will this tutorial be related to Kafka? If so, should we move them to `streaming/kafka`?

Reply:
Nope, sorry, my mistake, most of the code was copied from the Kafka tutorial. Will fix these kafkas today

Reply:
fixed